packages/slice-machine/src/components/SideNav/SideNav.css.ts

```diff
@@ -0,0 +1,346 @@
+import { colors, sprinkles, vars, selectors, mode } from "@prismicio/editor-ui";
+import { style } from "@vanilla-extract/css";
+
+// TODO: watch this PR https://github.com/vanilla-extract-css/vanilla-extract/pull/1105
+const block = style([
+  sprinkles({
+    boxSizing: "border-box",
+    fontFamily: "body",
+    all: "unset",
+  }),
+]);
+
+const blockWithDisplayRevert = style([
+  block,
+  sprinkles({
+    display: "revert",
+  }),
+]);
+
+const interactiveElement = style([
+  sprinkles({
+    transitionDuration: 200,
+    transitionTimingFunction: "easeOut",
+  }),
+  {
+    transitionProperty:
+      "background-color, border-color, boxShadow, fill, color",
```

Comment:
There is no transition in color, no? Meaning the color never change on hover or focus I think

Reply:
when it goes to active there's a color change for the text ad icons.

Reply:
You're right!